models/demos/llama3/demo/demo.py

```diff
@@ -146,35 +146,25 @@ def run_llama_demo_n300(user_input, batch_size, device_mesh, instruct_mode, is_c
         batch_prompts.append([input_prompts[(j + i) % len(input_prompts)] for j in range(len(input_prompts))])
 
     # Load model args, weights, and tokenizer
-    model_args = TtModelArgs(device_mesh.get_devices()[0], instruct=instruct_mode)
+    model_args = TtModelArgs(mesh_device, instruct=instruct_mode)
     tokenizer = Tokenizer(model_args.tokenizer_path)
 
-    model_args.n_layers = 32
```

Comment:
Remove from demo script?

Reply:
Yes. the default is always the max number of layers. this was just overriding it. Now we support any llama3 model and the n_layers differs, so we don't need this anymore.